pygmt/src/coast.py

```diff
@@ -200,7 +200,6 @@ def coast(self, **kwargs):
     ... )
     >>> # Show the plot
     >>> fig.show()
-    <IPython.core.display.Image object>
```

Comment:
`<IPython.core.display.Image object>` should be removed because `Figure.show()` returns nothing.